sponsor/models.py

```diff
@@ -24,3 +24,29 @@ def logo_display_for_admin(self):
             return "No logo"
 
     logo_display_for_admin.allow_tags = True
+
+
+class Donor(models.Model):
+    name = models.CharField(max_length=200, null=True, blank=True)
+    amount = models.FloatField()
+    visible = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.name
+
+
+    class Meta:
+        ordering = ("amount",)
+
+
+class Quote(models.Model):
+    name = models.CharField(max_length=200, null=True, blank=True)
+    quote = models.TextField()
+    picture = models.FileField()
```

Comment:
If we really want a picture (but I'm fine with removing it completely and just have a text & name in Quote alone), we could do that similarly to the: https://github.com/amakarudze/djangogirls/blob/d951efb7ae05548793a5b38cb7c3f00116b9218c/core/models.py#L118 using `ImageField`.

But to be fair, let's just remove it completely and not worry about picture - this will be extra work to choose them and make sure they are displayed nicely. I think we could just skip this step and prioritise having the page up and running.